### ISSUE TITLE
JAWS Screenreader not Reading TextArea Placeholder and InvalidText

### ISSUE THREAD

**Issue by embrown-ibm** [carbon-design-system/carbon#5547]
## Title line: 
JAWS Screenreader not Reading TextArea Placeholder and InvalidText.  As part of our accessibility testing using JAWS, the placeholder and the invalidText of our `<TextArea>` component are not being read.

## Environment

> Operating system
Windows 10

> Browser
Firefox

> Automated testing tool and ruleset

> Assistive technology used to verify
JAWS 2019

## Detailed description

> What version of the Carbon Design System are you using?
v10.8.0

> What did you expect to happen?
We expected the placeholder and invalidText would be read just as they are for the `<TextInput>` component.

> What happened instead?
The placeholder and invalidText is read for the `<TextInput>` component but not the `<TextArea>`.

## Steps to reproduce the issue

1. Add a `<TextInput>` to a dialog
2. Add a `<TextArea>` to a dialog
3. Open the dialog and attempt to read all of the text contained in it.

## Additional information

![image](https://user-images.githubusercontent.com/25669267/76011863-41a2d900-5ee3-11ea-8743-44de927bc5a1.png)

**Comment by tw15egan** [comment]
According to w3, placeholder text is not meant/able to be read by screen readers (https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text)

The invalid text not being read is a problem.

**Comment by dakahn** [comment]
Just so we're clear is the error text not accessible using JAWS whatsoever? like you can't use arrow down to read it at all -- or is it that it isn't being read when the error occurs. Thanks for reporting btw

**Comment by dakahn** [comment]
Confirmed this with JAWS 2020 on Firefox latest. The validation text is accessible via screen reader, but it is not read when the error is thrown.

**Comment by embrown-ibm** [comment]
@dakahn I noticed that you added the label `waiting for author's response`.  To what do you want me to respond?  This still appears to be a problem.  The problem was originally found during our 1Q Accessibility test and remained a problem during our 2Q Accessibility test.  Will there be a solution to this problem soon?

**Comment by dakahn** [comment]
No response needed, just a mis-click 👍

Can't give you an ETA, but I have plans to get to some backlogged JAWS a11y work in a coming sprint soon. Stay tuned

**Comment by dakahn** [comment]
update: we're going to knock out some JAWS issues next sprint. Our sprint planning doc reflects that here
https://github.com/carbon-design-system/carbon/issues/6772
